src/compiler/file_uploader/uploader_actor.ts

```diff
@@ -1,11 +1,12 @@
 import { Actor, ActorMethod, ActorSubclass } from '@dfinity/agent';
 import { createAuthenticatedAgent } from '../../../dfx';
+import { AZLE_UPLOADER_IDENTITY_NAME } from './uploader_identity';
 
 export type UploaderActor = ActorSubclass<_SERVICE>;
 
 export async function createActor(
     canisterId: string,
-    identityName?: string
+    identityName: string = AZLE_UPLOADER_IDENTITY_NAME
```

Comment:
Why do we default to this? Can we just make this required? Doesn't seem like we're using the default anywhere anyways?

Reply:
sure, we do use it in the tests... but I guess it's fine